metafix/src/test/java/org/metafacture/metafix/MetafixMethodTest.java

```diff
@@ -36,6 +38,9 @@
 @ExtendWith(MetafixToDo.Extension.class)
 public class MetafixMethodTest {
 
+    private static final String YOUTUBE_URL = "https://www.youtube.com/watch?v=daLgsPSvD9A";
+
+    private static final String YOUTUBE_URL_AS_URL_URL_BASE64 = "aHR0cHM6Ly93d3cueW91dHViZS5jb20vd2F0Y2g_dj1kYUxnc1BTdkQ5QQ==";
```

Comment:
Was this double `URL_URL` on purpose or is it a typo?

But with the adjusted test this constant is no longer necessary, anyway.

Reply:
Right, removed the variable. Force pushed, see https://github.com/metafacture/metafacture-fix/pull/386/commits/d1498c0ca64bb6ed57a6d97c05101fc8dc04d9cf  .